test-runner.js

```diff
@@ -158,25 +164,13 @@ function waitForOutput(process, matcher, timeout) {
     });
 }
 
-function setCustomCapabilities(appiumCapabilitiesLocation) {
-    var file = fs.readFileSync(appiumCapabilitiesLocation);
-    process.env.APPIUM_CAPABILITIES = file;
-    console.log("Custom capabilities found.");
-}
-
 function searchCustomCapabilities() {
-    var fileName = "appium.capabilities.json";
-    var projectDir = path.dirname(path.dirname(__dirname));
-    var pluginRootDir = path.dirname(projectDir);
-    var pluginAppiumCapabilitiesLocation = path.join(pluginRootDir, fileName);
-    var appAppiumCapabilitiesLocation = path.join(projectDir, fileName);
-    var customCapabilitiesLocation;
-
-    if(capabilitiesLocation){
-        customCapabilitiesLocation = path.join(projectDir, capabilitiesLocation, fileName);
-    }
+    const fileName = "appium.capabilities.json";
+    const pluginAppiumCapabilitiesLocation = utils.resolve(pluginRoot, fileName);
```

Comment:
pluginRoot here is different from what we expect to be. It should be the the parent of 'demo' folder. This is exactly what path.dirname(projectDir) returns